main.cpp

```diff
@@ -65,9 +64,27 @@ void variableDeclarations()
     //example:
     int number = 2; //declaration of a variable named "number", that uses the primitive type 'int', and the variable's initial value is '2'
     
-
+    int cash = 0;
+    int level = 0;
+    int point = 0;
+
+    float limit = 0.0;
+    float height = 0.0;
+    float width = 0.0;
```

Comment:
Initialize floats with floats. You have written doubles on the right side of the `=`
- [ ] revised